repair/repair.hh

```diff
@@ -496,6 +496,15 @@ enum class node_ops_cmd : uint32_t {
      replace_heartbeat,
      replace_abort,
      replace_done,
+     decommission_prepare,
+     decommission_heartbeat,
+     decommission_abort,
+     decommission_done,
+     bootstrap_prepare,
+     bootstrap_heartbeat,
+     bootstrap_abort,
+     bootstrap_done,
+     query_pending_ops,
```

Comment:
I do not know if this is supposed. Since we need to modify idl files very  carefully than internal headers. Leaving it as a separate files makes more sense to me.  idl file has its own syntax, even if it looks similar to .hh file.

Reply:
(the plan is to use C++ reflection to create serializers/deserializers in C++ instead of idl-compiler.py)

https://en.cppreference.com/w/cpp/keyword/reflexpr